hbase-server/src/main/java/org/apache/hadoop/hbase/util/LossyCounting.java

```diff
@@ -119,16 +138,33 @@ public long getDataSize() {
     return data.size();
   }
 
-  public boolean contains(String key) {
+  public boolean contains(T key) {
     return data.containsKey(key);
   }
 
-  public Set<String> getElements(){
+  public Set<T> getElements(){
     return data.keySet();
   }
 
   public long getCurrentTerm() {
     return currentTerm;
   }
+
+  class SweepRunnable implements Runnable {
+    @Override public void run() {
+      if (LOG.isTraceEnabled()) {
+        LOG.trace("Starting sweep of lossyCounting-" + name);
```

Comment:
nit, could use the `{}` markers and drop the `isTraceEnabled()`

Reply:
This came in from branch-2 change. I agree but try not to second guess the original contributor on backports.